modules/delegates/components/DelegateCard.tsx

```diff
@@ -83,11 +83,28 @@ export function DelegateCard({ delegate }: PropTypes): React.ReactElement {
                 <DelegatePicture delegate={delegate} />
 
                 <Box sx={{ ml: 2 }}>
-                  <Box>
+                  <Flex sx={{ alignItems: 'center' }}>
                     <Text as="p" variant="microHeading" sx={{ fontSize: [3, 4] }}>
-                      {delegate.name ? limitString(delegate.name, 43, '...') : 'Unknown'}
+                      {delegate.name
+                        ? limitString(delegate.name, isOwner ? 23 : 43, '...')
+                        : limitString('Unknown', isOwner ? 12 : 43, '...')}
                     </Text>
-                  </Box>
+                    {isOwner && (
+                      <Flex
+                        sx={{
+                          display: 'inline-flex',
+                          backgroundColor: 'tagColorSevenBg',
+                          borderRadius: '12px',
```

Comment:
just fyi we do have`'roundish'` which equals 12px for `borderRadius` https://github.com/makerdao/dai-ui/blob/master/packages/dai-ui-theme-maker-neue/lib/index.js#L85

Reply:
Good call. Updated.